tests/e2e/utils/shared.ts

```diff
@@ -137,6 +142,11 @@ export const createSingleType = async (page, data) => {
 };
 
 export const createCollectionType = async (page, data) => {
+  // TODO: fix webkit bug with selecting ctb attributes
+  if (page.context()._browser._name === 'webkit') {
+    return test.fixme();
+  }
```

Comment:
```suggestion
  if (page.context()._browser._name === 'webkit') {
    return test.fixme();
  }
```

const browserName = page.context().browser().browserType().name();
if (browserName === 'webkit') {
    return test.fixme();
}

Reply:
This block is temporary and copy-pasted several places so I'll leave this one alone